### PR TITLE
Cart Screen

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Cart Screen

recording

### CODE REVIEW

File: screens/cart.html [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+            <li class="navbar-link">
+              <a href="#">About Us</a>
+            </li>
+            <li class="navbar-link">
+              <a href="product-listing.html">Products</a>
+            </li>
+          </ul>
+        </section>
+        <section class="navbar-actions">
+          <!-- <span class="material-icons navbar-action navbar-icon"> search </span> -->
```
Comment (AtulHande03a) [inline screens/cart.html:39]: remove comments if not necessary

File: screens/cart.html [hunk 2]
```diff
@@ -0,0 +1,295 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (AtulHande03a) [inline screens/cart.html:6]: add author meta tag

**Review by AtulHande03a — APPROVED** [review]
apart from nominal comments rest looks MASTA!!

File: screens/cart.html (hunk 1 above)
Comment (horsemaker) [reply screens/cart.html:39]: I plan to add it if I get time. 🥺

File: stylesheets/cart.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  display: flex;
+  gap: 6rem;
+  justify-content: center;
+}
+
+.cart-active {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
```
Comment (t007rushi) [inline stylesheets/cart.css:14]: this can be added to utility

File: stylesheets/cart.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  gap: 0.5rem;
+}
+
+.card-price-original {
+  text-decoration: line-through;
+}
+
+.card-discount {
+  color: var(--teal);
```
Comment (t007rushi) [inline stylesheets/cart.css:37]: what does teal signify? make it teal-color simplified

File: stylesheets/cart.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  text-transform: uppercase;
+  font-weight: var(--semi-bold);
+}
+
+.cart-checkout {
+  min-width: 20rem;
+  position: sticky;
+  top: 7.5rem;
+  align-self: flex-start;
+  background-color: #fefefe;
```
Comment (t007rushi) [inline stylesheets/cart.css:80]: color variables

File: stylesheets/cart.css [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+.checkout-delivery {
+  font-weight: var(--semi-bold);
+}
+
+.checkout-amount {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```
Comment (t007rushi) [inline stylesheets/cart.css:120]: this also can be utility just suggestion

File: stylesheets/cart.css [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+.cart-checkout {
+  min-width: 20rem;
+  position: sticky;
+  top: 7.5rem;
+  align-self: flex-start;
+  background-color: #fefefe;
+  border-radius: 5px;
+  padding: 1rem;
+  box-shadow: #63636333 0px 1px 4px 0px;
```
Comment (t007rushi) [inline stylesheets/cart.css:83]: same as above

**Review by t007rushi — APPROVED** [review]
UI looks refreshing

File: screens/cart.html (hunk 2 above)
Comment (horsemaker) [reply screens/cart.html:6]: Didn't know about this. Thank you!

File: stylesheets/cart.css (hunk 3 above)
Comment (horsemaker) [reply stylesheets/cart.css:14]: Plan to add this to my component library later. 👍

File: stylesheets/cart.css (hunk 4 above)
Comment (horsemaker) [reply stylesheets/cart.css:37]: Good Suggestion! Will add this in a misc Pull Request later for sure.

File: stylesheets/cart.css (hunk 5 above)
Comment (horsemaker) [reply stylesheets/cart.css:80]: Will add this in misc Pull Request

File: stylesheets/cart.css (hunk 7 above)
Comment (horsemaker) [reply stylesheets/cart.css:83]: right!

**Pull request merged**